Formula/med-file.rb

```diff
@@ -0,0 +1,48 @@
+class MedFile < Formula
+  desc "Modeling and Data Exchange standardized format"
+  homepage "http://www.salome-platform.org"
+  url "http://files.salome-platform.org/Salome/other/med-3.2.0.tar.gz"
+  sha256 "d52e9a1bdd10f31aa154c34a5799b48d4266dc6b4a5ee05a9ceda525f2c6c138"
+
+  depends_on "cmake" => :build
+  depends_on "gcc" => :build
+  depends_on "swig" => :build
+  depends_on "hdf5"
+  depends_on "python"
+
+  patch do
+    url "https://raw.githubusercontent.com/Homebrew/formula-patches/720fedf/med-file/libc%2B%2B_and_python_bindings_fixes.diff"
+    sha256 "4125238ca3623e682b766cf2d34fed17938007ad326ae3b158a9cd427c638054"
+  end
+
+  def install
+    python_prefix=`#{Formula["python"].opt_bin}/python2-config --prefix`.chomp
+    python_include=Dir["#{python_prefix}/include/*"].first
+    python_library=Dir["#{python_prefix}/lib/libpython*.dylib"].first
+
+    system "cmake", ".", "-DMEDFILE_BUILD_PYTHON=ON",
+                         "-DMEDFILE_BUILD_TESTS=OFF",
+                         "-DMEDFILE_INSTALL_DOC=ON",
+                         "-DPYTHON_INCLUDE_DIR=#{python_include}",
+                         "-DPYTHON_LIBRARY=#{python_library}",
+                         *std_cmake_args
+
+    system "make", "install"
+  end
+
+  test do
+    assert_match "Nombre de parametre incorrect : medimport filein [fileout]",
+                 shell_output("#{bin}/medimport 2>&1", 255)
+    (testpath/"test.c").write <<-EOS.undent
+      #include <med.h>
+      int main() {
+        med_int major, minor, release;
+        return MEDlibraryNumVersion(&major, &minor, &release);
+      }
+    EOS
+    system ENV.cc, "test.c", "-I#{include}",
+                   "-I#{Formula["hdf5"].opt_include}",
```

Comment:
this'll fit on the previous line

Reply:
I wrapped due to the 80-column guideline, wrapped at consecutive includes.

Reply:
it is 80 not 81

Reply:
fixed.